### PR TITLE
docs: rename some leftover from bindings and docs

### PR DESCRIPTION
**Pull request by kristifidani**
# Pull Request Template

## Motivation and Context

This PR attempts to fix / update / rename whatever we might have missed from the last series of renaming.

## Summary

Provide a short summary of changes in this pull request.

- Update api reference links for android bindings.
- Cleanup swift and wasm docs.
- Updating some links in general.

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

File: sdk/overrides/main.html [hunk 1]
```diff
@@ -134,8 +134,8 @@
 
             <!-- Middle section (your custom links) -->
             <div class="md-footer-custom-links">
-                <a href="https://cawaena.com/#/main/imprint">Imprint</a> |
-                <a href="https://cawaena.com/#/main/privacy">Privacy Policy</a>
+                <a href="https://cawaena.com/#/main/imprint">Imprint</a> | <!-- TODO: update this link? -->
```
Comment (hauju) [inline sdk/overrides/main.html:137]: https://etospheres.com/impressum.html

File: sdk/overrides/main.html [hunk 2]
```diff
@@ -134,8 +134,8 @@
...
+                <a href="https://cawaena.com/#/main/privacy">Privacy Policy</a> <!-- TODO: update this link? -->
```
Comment (hauju) [inline sdk/overrides/main.html:138]: https://etospheres.com/datenschutz.html

**Review by hauju — COMMENTED** [review]
Added the links for etoshperes as comment.

File: sdk/overrides/main.html (hunk 1 above)
Comment (antbern) [reply sdk/overrides/main.html:137]:
```suggestion
                <a href="https://etospheres.com/impressum.html">Imprint</a> |
```

File: sdk/overrides/main.html (hunk 2 above)
Comment (antbern) [reply sdk/overrides/main.html:138]:
```suggestion
                <a href="https://etospheres.com/datenschutz.html">Privacy Policy</a>
```

**Review by antbern — APPROVED** [review]

**Review by hauju — APPROVED** [review]

**Review by antbern — APPROVED** [review]
